core/src/impl/Kokkos_MemoryPool.cpp

```diff
@@ -42,9 +42,11 @@
 //@HEADER
 */
 
+#include <impl/Kokkos_Error.hpp>
+#include <Kokkos_HostSpace.hpp>
```

Comment:
Why do you need this file?

Reply:
you're right; neither are necessary

Reply:
Actually we do need `<impl/Kokkos_Error.hpp>`